core/__init__.py

```diff
@@ -34,6 +36,18 @@
 from viur.core import logging as viurLogging  # Initialize request logging
 
 
+def load_indexes_from_file():
+    try:
+        with open(os.path.join(utils.coreBasePath, 'index.yaml'), 'r') as file:
+            indexes = yaml.safe_load(file)
+            indexes = indexes.get("indexes", [])
+            for index in indexes:
+                index["properties"] = [_property["name"] for _property in index["properties"]]
+    except FileNotFoundError as e:
+        logging.error("index.yaml not found")
+        return
```

Comment:
Is this case really an error?
Or better have a warning, and return something empty which doesn't raise a `TypeError: argument of type 'NoneType' is not iterable` behind the call below?
```suggestion
        logging.warning("index.yaml not found")
        return {}
```